test/system/groups/samples_test.rb

```diff
@@ -767,5 +767,75 @@ def retrieve_puids
         click_on I18n.t('shared.samples.metadata.file_imports.errors.ok_button')
       end
     end
+
+    test 'can update metadata value that is not from an analysis' do
```

Comment:
Could you add a test case for a user who doesn't have permission to edit the metadata